vendor/bat-native-ledger/src/bat/ledger/internal/common/bind_util.cc

```diff
@@ -12,6 +12,58 @@
 
 namespace braveledger_bind_util {
 
+ledger::RewardsType ConvertIntToRewardsType(int type) {
+  ledger::RewardsType rewards_type = static_cast<ledger::RewardsType>(type);
+
+  switch (rewards_type) {
+    case ledger::RewardsType::AUTO_CONTRIBUTE:
+    case ledger::RewardsType::ONE_TIME_TIP:
+    case ledger::RewardsType::RECURRING_TIP: {
+      break;
+    }
+    default: {
+      rewards_type = ledger::RewardsType::UNKNOWN;
+    }
+  }
+  return rewards_type;
+}
+
+ledger::PromotionType ConvertIntToPromotionType(int type) {
+  ledger::PromotionType promotion_type =
+      static_cast<ledger::PromotionType>(type);
+
+  switch (promotion_type) {
+    case ledger::PromotionType::UGP:
+    case ledger::PromotionType::ADS: {
+      break;
+    }
+    default: {
+      promotion_type = ledger::PromotionType::UNKNOWN;
+    }
+  }
+  return promotion_type;
+}
+
+ledger::PromotionStatus ConvertIntToPromotionStatus(int type) {
+  ledger::PromotionStatus promotion_status =
+      static_cast<ledger::PromotionStatus>(type);
+
+  switch (promotion_status) {
+    case ledger::PromotionStatus::ACTIVE:
+    case ledger::PromotionStatus::ATTESTED:
+    case ledger::PromotionStatus::CLAIMED:
+    case ledger::PromotionStatus::SIGNED_TOKENS:
+    case ledger::PromotionStatus::FINISHED:
+    case ledger::PromotionStatus::OVER: {
+      break;
+    }
+    default: {
+      promotion_status = ledger::PromotionStatus::UNKNOWN;
+    }
+  }
+  return promotion_status;
+}
+
```

Comment:
These 'ConvertIntTo' enum helpers might be better as more accessible methods somewhere else?